cmd/oras/internal/option/pusher.go

```diff
@@ -73,3 +93,22 @@ func decodeJSON(filename string, v interface{}) error {
 	defer file.Close()
 	return json.NewDecoder(file).Decode(v)
 }
+
+// getAnnotationsMap get resharp annotationslice to target type
+func getAnnotationsMap(ManifestAnnotationSlice []string, annotations map[string]map[string]string) error {
+	re := regexp.MustCompile(`=\s*`)
+	rawAnnotationsMap := make(map[string]string)
+	for _, rawAnnotation := range ManifestAnnotationSlice {
+		rawAnnotation := re.Split(rawAnnotation, 2)
+		rawAnnotation[0], rawAnnotation[1] = strings.TrimSpace(rawAnnotation[0]), strings.TrimSpace(rawAnnotation[1])
+		if annotationLength := len(rawAnnotation); annotationLength != 2 {
+			return errors.New("invalid annotation")
+		}
```

Comment:
```suggestion
	        parts := strings.SplitN(raw, "=", 2)
	        if len(parts) < 2 {
		        // return invalid
	        }
	        key, value := strings.TrimSpace(parts[0]), strings.TrimSpace(parts[1])
```

Reply:
If annotation key doesn't contain `=`, we can use https://pkg.go.dev/strings#SplitN to simplify the parsing

Reply:
ACK